libs/app-shell/src/lib/components/plugin-launcher/plugin-launcher.component.ts

```diff
@@ -61,22 +52,24 @@ export class PluginLauncherComponent implements OnChanges {
             }
             this.descriptor = item.descriptor;
         }
-        this.doCreateComponent(this.descriptor);
+        this.doCreateComponent(this.descriptor, this.module);
     }
 
 
-    async doCreateComponent(descriptor: Partial<PluginDescriptor>): Promise<void> {
-        const component = await loadRemoteModule(descriptor)
-            .then(m => m[descriptor.componentName]);
-        const factory = this.cfr.resolveComponentFactory(component);
+    async doCreateComponent(descriptor: Partial<PluginDescriptor>, moduleName: string): Promise<void> {
+        const _module = descriptor.modules.find(module => module.name === moduleName);
+        if (_module.type === 'angular-ivy-component') {
```

Comment:
How do we have webcomponents ? When I used plugin launcher to execute all , and I had little IF ELSE that works with webcomponents

Reply:
I will be adding it tomorrow. Iframe, web components...

Reply:
ok